### PR TITLE
seqGermline implementation

### PR DESCRIPTION
**Pull request by Alexander230**
Germline creation step for BCR pipeline

### CODE REVIEW

File: R/germline.R [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+          as_tibble() %>%
+          germline_single_df() %>%
+          return()
+      }) %>%
+      return()
+  } else {
+    .data %>%
+      as_tibble() %>%
+      germline_single_df() %>%
+      return()
```
Comment (tsvvas) [inline R/germline.R:53]: Did you check this with all data formats immunarch works with? Including db connections

**Review by tsvvas — APPROVED** [review]

File: R/germline.R (hunk 1 above)
Comment (Alexander230) [reply R/germline.R:53]: Do we have guides how to test it locally with different formats? Here I must convert data to dataframe to merge with reference dataframe; I think if as_tibble() doesn't work with some of the supported formats, we need to make common conversion function in tools.R.

Comment (Alexander230) [reply R/germline.R:53]: Found data formats in our tutorial: https://immunarch.com/articles/v2_data.html
It mentions MonetDBLite, which is deprecated and removed from CRAN; modern analog is duckDB. Both duckDB and Apache Spark can provide access with `tbl` function from `dplyr`; and this table can be converted to dataframe with `as_tibble()`, so this function will work with `dplyr::tbl()` compatible databases.

**Review by ivan-immunomind — APPROVED** [review]
it is hard to check add_column_without_alleles regexp's, but hope that all is ok

**Pull request merged**